src/containers/trending-playlists/store/lineups/collections/reducer.ts

```diff
@@ -4,7 +4,8 @@ import { PREFIX } from './actions'
 
 export const initialState = {
   ...initialLineupState,
-  prefix: PREFIX
+  prefix: PREFIX,
+  maxEntries: 30
```

Comment:
why is this 30? is that what the backend does? maybe add a comment for when we rework this

Reply:
Yeah, playlist trending is limited to 30 on the backend for perf reasons